CRM/Member/Form/Task/Label.php

```diff
@@ -133,7 +133,8 @@ public function postProcess() {
         'return' => 'contact_id',
       ));
       foreach ($memberships['values'] as $id => $membership) {
-        $labelRows[$id] = $rows[$membership['contact_id']];
+        if (isset($rows[$membership['contact_id']]))
+          $labelRows[$id] = $rows[$membership['contact_id']];
```

Comment:
@liedekef you need to provide curly braces around the code because "Inline control structures are not allowed" as per the style warning https://test.civicrm.org/job/CiviCRM-Core-PR/11113/checkstyleResult/new/